.claude/skills/backlog-refinement/SKILL.md

```
@@ -0,0 +1,273 @@
+gh issue edit <issue> --add-blocked-by <other-issue>
+gh issue edit <issue> --add-blocking <other-issue>
+```
+
+For strict parent/sub-issue decomposition (splitting one oversized issue
+into several trackable pieces): `gh issue edit <parent> --add-sub-issue
+<child-number-or-url>`.
+
+Native links are structural, not explanatory — they show _that_ two issues
+are related but not _why_. Pair every native `--add-blocked-by`/
+`--add-blocking` link with a rationale comment, the same shape used for
+#13 ↔ #31 today (recorded in prose only, since it predates this skill's use
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)